devices/debian_wifi.py

```diff
@@ -53,3 +54,38 @@ def wlan_connectivity(self):
     def disconnect_wpa(self):
         self.sendline("killall wpa_supplicant")
         self.expect(self.prompt)
+
+    def wlan_ssid_connect(self, ssid_name):
+        self.sendline("iwconfig %s essid %s" % (self.iface_wlan,ssid_name))
+        self.expect(self.prompt)
+
+    def wlan_ssid_disconnect(self):
+        self.sendline("iw dev %s disconnect" % self.iface_wlan)
+        self.expect(self.prompt)
+
+    def start_lan_client(self, wan_gw=None):
+        self.iface_dut = self.iface_wlan
```

Comment:
Do we need iface_wlan at all? Should just use iface_dut instead?

Reply:
If I use iface_dut it is taking eth0 as default. As I wasn't the interface to be wlan1 am passing the iface_wlan1 here